modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
@dkarlovi thanks for explaining more. To ease the migration, I think we will need to provide auto-generated documentation on how to set these values using Guzzle (the default PSR-7 http client)